File: 217-contains-duplicate/217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> set = new  HashSet<Integer>();
        int n = nums.length;
        for(int i=0;i<n;i++){
            set.add(nums[i]);
        }
        if(set.size()<n)
        {
            return true;
        }
        return false;
    }
}

File: 217-contains-duplicate/217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> dp;
        for(int n:nums){
            dp.insert(n);
        }
        if(dp.size()<nums.size())
            return true;
        
        return false;
        
    }
};